docs/UserGuide.adoc

```diff
@@ -54,10 +54,101 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 === Recipe inventory commands
 
 === Database commands
+==== View a recipe from the database: `cookbook view recipe`
+- View a recipe, its ingredients, and the steps to cook it. +
```

Comment:
Are we using bullets? We should probably standardise the use of bullets, and also whether we use "+" or newline as line break, can discuss further. I personally think bullets aren't necessary, but changing the line breaks within one command to "+" and newline between commands seems fine. Also might need to consider how we want to do line break for tips.

Reply:
Let's use lists (-) for the format and examples, and the actual examples will be sub-lists (*). Newlines will be indicated using 2 newlines rather than +, since that looks clearer in the adoc as well?

Reply:
alright!